selfdrive/ui/qt/widgets/scroller.cc

```diff
@@ -0,0 +1,17 @@
+#include "scroller.hpp"
+
+GoodScrollArea::GoodScrollArea(QWidget *area){
+  setWidget(area);
+  setWidgetResizable(true);
+  setVerticalScrollBarPolicy(Qt::ScrollBarAlwaysOff);
+  setHorizontalScrollBarPolicy(Qt::ScrollBarAlwaysOff);
+  setStyleSheet("background-color:transparent;");
+
+  QScroller *scroller = QScroller::scroller(this->viewport());
+  QScrollerProperties sp = scroller->scrollerProperties();
+
+  sp.setScrollMetric(QScrollerProperties::VerticalOvershootPolicy, QVariant::fromValue<QScrollerProperties::OvershootPolicy>(QScrollerProperties::OvershootAlwaysOff));
+
+  scroller->grabGesture(this->viewport(), QScroller::TouchGesture);
```

Comment:
we use `QScroller::LeftMouseButtonGesture` currently. does `QScroller::TouchGesture` work the same?